packages/customize-widgets/src/components/customize-widgets/use-clear-selected-block.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+export default function useClearSelectedBlock( sidebarControl, popoverRef ) {
```

Comment:
Could you please add a comment explaining why we can't use `BlockSelectionClearer`?